music/src/core/data.ts

```diff
@@ -799,3 +800,170 @@ export class MultitrackConverter extends DataConverter {
     return noteSequence;
   }
 }
+
+/**
+ * Converts to and from hit/velocity/offset representations.
+ * In this setting, we represent drum sequences and performances
+ * as triples of (hit, velocity, offset). Each timestep refers to a fixed beat
+ * on a grid, which is by default spaced at 16th notes (when `stepsPerQuarter`
+ * is 4). Drum hits that don't fall exactly on beat are represented through the
+ * offset value, which refers to the relative distance from the nearest
+ * quantized step.
+ *
+ * Hits are binary [0, 1].
+ * Velocities are continuous values in [0, 1].
+ * Offsets are continuous values in [-0.5, 0.5], rescaled to [-1, 1] for
+ * tensors.
+ *
+ * @param stepsPerQuarter The number of quantization steps per quarter note.
+ * @param humanize If True, flatten all input velocities and microtiming. The
```

Comment:
hmm this arg name and its description are opposite to me. Like, humanizing to me means "with microtiming", not without

Reply:
Actually, what are these arguments that you're documenting to? If it's to the `GrooveConverterArgs`, `flattenVelocities` isn't one of them, and there's 2 other arguments that aren't documented.

Reply:
The argument description was more relevant to the training model. Updated here to clarify.

Reply:
Where are you seeing flattenVelocities?